src/main/java/de/tum/in/www1/artemis/service/connectors/LtiService.java

```diff
@@ -311,20 +264,19 @@ private void addUserToExerciseGroup(User user, Course course) {
             user.setGroups(groups);
             userCreationService.saveUser(user);
 
-            if (!user.getLogin().startsWith("edx")) {
-                // try to sync with authentication service for actual users (not for edx users)
-                try {
-                    artemisAuthenticationProvider.addUserToGroup(user, courseStudentGroupName);
-                }
-                catch (ArtemisAuthenticationException e) {
-                    // This might throw exceptions, for example if the group does not exist on the authentication service. We can safely ignore it
-                }
+            // TODO: is there an issue with removing the old edx check?
```

Comment:
Open Todo comment.

Reply:
It was an old TODO, I removed it now, thanks